### PR TITLE
[KYUUBI #1329] Support download Flink binary package in kyuubi-download module

### PR DESCRIPTION
**Pull request by yanghua**
…

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: docs/develop_tools/building.md [hunk 1]
```diff
@@ -73,21 +73,21 @@ Profile | Default  | Since
 -Pspark-3.1 | No | 1.1.0
 
 
-## Defining the Apache Mirror for Spark
+## Defining the Apache Mirror for Spark or Flink
 
-By default, we use `https://archive.apache.org/dist/spark/` to download the built-in Spark release package,
+By default, we use `https://archive.apache.org/dist/spark/` to download the built-in Spark or Flink release package,
```
Comment (pan3793) [inline docs/develop_tools/building.md:78]: url

File: docs/develop_tools/building.md [hunk 2]
```diff
@@ -73,21 +73,21 @@ Profile | Default  | Since
...
 ```bash
 build/mvn clean package -Dspark.archive.mirror=https://mirrors.bfsu.edu.cn/apache/spark/spark-3.0.1
 ```
 
 Visit [Apache Mirrors](http://www.apache.org/mirrors/) and choose a mirror based on your region.
 
-Specifically for developers in China mainland, you can use the pre-defined profile named `mirror-cn` which use
-`mirrors.bfsu.edu.cn` to speed up Spark Binary downloading. For instance,
+Specifically for developers in China mainland, you can use the pre-defined profile named `spark-mirror-cn` 
+or `flink-mirror-cn` which use `mirrors.bfsu.edu.cn` to speed up Spark Binary downloading. For instance,
```
Comment (pan3793) [inline docs/develop_tools/building.md:89]: `mirror-cn` for two is enough?

Comment (yanghua) [reply docs/develop_tools/building.md:89]: OK~

File: pom.xml [hunk 3]
```diff
@@ -123,6 +123,10 @@
         <spark.archive.name>spark-${spark.version}-bin-hadoop${hadoop.binary.version}.tgz</spark.archive.name>
         <spark.archive.mirror>https://archive.apache.org/dist/spark/spark-${spark.version}</spark.archive.mirror>
         <spark.archive.download.skip>false</spark.archive.download.skip>
+        <flink.version>1.12.5</flink.version>
```
Comment (pan3793) [inline pom.xml:126]: to L100, sort alphabetically

**Review by pan3793 — APPROVED** [review]

**Review by ulysses-you — APPROVED** [review]

**Comment by ulysses-you** [comment]
thanks, merging to master

**Pull request closed without merging**